data/virt_autotest/guest_params_xml_files/sles_15_sp5_64_kvm_hvm_uefi_x86_64_milestone.xml

```diff
@@ -0,0 +1,93 @@
+<?xml version="1.0"?>
+<guest>
+    <guest_os_name>sles</guest_os_name>
+    <guest_version>15-sp5</guest_version>
```

Comment:
All LGTM, except one open question that whether we should enable sle15sp5 guest or sle15sp4 guest in micro 5.4 test. Enabling developing (15sp5) guest in developing product's (micro 5.4) test is unusual.  I will ask Jan about it.

Reply:
Got reply from Jan -- we test sle15sp4 as vm until 15sp5 is released, then switch to sp5.

Reply:
Done